manipulation/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
My method was to only add those lines when catkin/gcc complained, but I think you're right I should have been consistent across the whole repo. Note that there are other MoveIt! repos that also have not been switched yet (e.g. moveit_plugins). Should I fix this repo fully?